src/Glimpse.Common/Broker/MessageContext.cs

```diff
@@ -9,6 +9,7 @@ public class MessageContext
 
         public string Type { get; set; }
 
+        [JsonIgnore]
```

Comment:
I thought about putting on the broker itself but that doesn't seem right either. Maybe there is a "context" property that has what is current the messagecontext and this operationcontext as properties?